yasa/hypno.py

```diff
@@ -328,6 +328,16 @@ def scorer(self):
 
     # CLASS METHODS BELOW
 
+    def copy(self):
+        """Return a new copy of the current Hypnogram."""
+        return type(self)(
```

Comment:
Is there an advantage of using `type(self)` rather than `Hypnogram`?

Reply:
I suppose it's more flexible but it doesn't really offer any benefit here.